.buildkite/scripts/steps/test/scout_test_run_builder.sh

```diff
@@ -8,7 +8,23 @@ source .buildkite/scripts/common/util.sh
 .buildkite/scripts/copy_es_snapshot_cache.sh
 
 echo '--- Discover Playwright Configs and upload to Buildkite artifacts'
-node scripts/scout discover-playwright-configs --save
+
+set +e;
+OUTPUT=$(node scripts/scout discover-playwright-configs --save --validate 2>&1)
```

Comment:
When this moves to on-merge it may make sense to move this to quick checks.   It'll fail CI quicker if there's issues